app/assets/stylesheets/wpcc/section/_contributions.scss

```diff
@@ -1,3 +1,7 @@
+.contributions__content {
```

Comment:
At a later stage I will modify this slightly to hide the contents and display if the step is active:

```
display: none;

&.is-active {
  display: block;
}
```